src/cmake/macros/FindMySQL.cmake

```diff
@@ -143,14 +143,14 @@ find_path(MYSQL_INCLUDE_DIR
     /usr/local/include
     /usr/local/include/mysql
     /usr/local/mysql/include
+    "C:/Program Files/MySQL/MySQL Server 8.0/include"
     "C:/Program Files/MySQL/MySQL Server 5.7/include"
-    "C:/Program Files/MySQL/MySQL Server 5.6/include"
```

Comment:
is deprecating MySQL 5.6 needed in order to support 8.0? if yes, ok for me. Otherwise we can keep it until February 2021

Reply:
No, but if I understand what I have read correctly, these changes can't be supported by 5.6.

But if someone have 5.6 and can test the changes without deprecating it, feel free. I cannot test that myself since I fully upgraded to 8.0

Reply:
8.0 breaks 5.6 its true

Reply:
Yea that's what I thought too after what I read online, but wasn't too sure!
Thank you for confirming!!

Reply:
ok, then it's fine for me. Supporting 8.0 it's better than 5.6. Also @Viste let us know if it's ok for Maria too, because that's even more important